cores/esp8266/HardwareSerial.cpp

```diff
@@ -133,8 +142,8 @@ unsigned long HardwareSerial::detectBaudrate(time_t timeoutMillis)
     return detectedBaudrate;
 }
 
-size_t HardwareSerial::readBytes(char* buffer, size_t size)
-{
+size_t HardwareSerial::readBytes(char* buffer, size_t size)

```

Comment:
Is this a linefeed change here?  I think this whole file is DOS \r\n formatted, but your patch only has \UNIX n.  For now, could you make sure you match the linefeeds of the rest of the file (yes, I hate DOS linefeeds, too, but we need to be consistent in any particular file).

Reply:
I really do not know. I did not touch these lines. Could you please check it?

Reply:
It is indeed a crlf change.
And after checking, this PR restores consistency which is not here in current master.
(The Allman process shall have a try-again session)